bootstrap-cpp.sh

```diff
@@ -29,6 +29,5 @@ cmake3 \
     -DEXTRA_LINK_DIRECTORY="/opt/vespa-boost/lib;/opt/vespa-libtorrent/lib;/opt/vespa-zookeeper-c-client/lib;/opt/vespa-cppunit/lib;/usr/lib64/llvm3.9/lib" \
     -DEXTRA_INCLUDE_DIRECTORY="/opt/vespa-boost/include;/opt/vespa-libtorrent/include;/opt/vespa-zookeeper-c-client/include;/opt/vespa-cppunit/include;/usr/include/llvm3.9" \
     -DCMAKE_INSTALL_RPATH="/opt/vespa/lib64;/opt/vespa-boost/lib;/opt/vespa-libtorrent/lib;/opt/vespa-zookeeper-c-client/lib;/opt/vespa-cppunit/lib;/usr/lib/jvm/java-1.8.0/jre/lib/amd64/server;/usr/include/llvm3.9" \
-    -DCMAKE_BUILD_RPATH=/opt/vespa/lib64 \
     -DVALGRIND_UNIT_TESTS=no \
```

Comment:
This line (setting VALGRIND_UNIT_TESTS to no)  isn't needed either, default is FALSE, cf. build_settings.cmake.